example_case_folders/07_floris_standin_and_solar_pysam/hercules_input_Flatirons.yaml

```diff
@@ -42,8 +42,8 @@ py_sims:
 
     py_sim_type: SolarPySAM
     verbose: True # flag to print debugging statements
-    weather_file_name: Flatirons_solar_data_precursor-tztest.csv
-    system_info_file_name: 100MW_1axis_pvsamv1.json
+    weather_file_name: inputs/Flatirons_solar_data_highDNI.csv
```

Comment:
trimmed input solar data file to shift the simulation to a time period with higher DNI so that solar power output is much greater than zero